bundles/pixi.js-node/src/index.ts

```diff
@@ -10,10 +10,7 @@ import { NoiseFilter } from '@pixi/filter-noise';
 import '@pixi/mixin-cache-as-bitmap';
 import '@pixi/mixin-get-child-by-name';
 import '@pixi/mixin-get-global-position';
-// eslint-disable-next-line @typescript-eslint/no-duplicate-imports
 import { NodeCanvasResource } from './adapter';
```

Comment:
The `import './adapter';` and `import { NodeCanvasResource } from './adapter';` is manually changed. It is not necessary to import adapter ahead of everything in my opinion.